docs/tutorial/part-one/index.md

```diff
@@ -343,7 +343,7 @@ The Gatsby `<Link />` component is for linking between pages within your site. F
 
 ## Deploying a Gatsby site
 
-Gatsby.js is a _static site generator_, which means there are no servers to setup or complicated databases to deploy. Instead, the Gatsby `build` command produces a directory of static HTML and JavaScript files which you can deploy to a static site hosting service.
+Gatsby.js is a _static site and PWA (Progressive Web App) generator_, which means there are no servers to setup or complicated databases to deploy. Instead, the Gatsby `build` command produces a directory of static HTML and JavaScript files which you can deploy to a static site hosting service.
```

Comment:
I don't think this is necessarily true, a static site can be a PWA, a PWA is not necessarily a static site. And why is this distinction important? Seems to bloat the tag line

Reply:
Thanks for your thoughts, it's not necessarily about describing the outputs (static site or PWA) - hence starting a convo and admitting the tag-line would need work - I absolutely agree with you that its too bloated.

I think the important point for me is that we evolve and grow how we describe Gatsby. Describing it as a simply a static site generator does it a massive disservice. 

**A blazing fast web app generator** might be more simple, but again does it say enough?

Reply:
Thanks @dominicfallows! Getting this messaging right is something that's currently being wrestled with :) Check out the home page on [next.gatsbyjs.org](https://next.gatsbyjs.org/) for another iteration of the message. 

Could you update your PR to use the following tagline instead?

> Blazing fast modern site generator for React.js

Hopefully that's concise enough to cover what Gatsby is, without cornering it as a plain SSG.